exchanges/huobi/huobi_futures.go

```diff
@@ -90,11 +93,11 @@ func (h *HUOBI) FGetContractInfo(ctx context.Context, symbol, contractType strin
 	if symbol != "" {
 		params.Set("symbol", symbol)
 	}
-	if contractType != "" {
-		if !common.StringSliceCompareInsensitive(validContractTypes, contractType) {
-			return resp, errors.New("invalid contractType")
+	if t := strings.ToLower(contractType); t != "" {
```

Comment:
You'd make an empty string lowercase to avoid an extra line of code 😄 no change required

Reply:
Note: If it was a hotter path I wouldn't do this.
I sniffed the cost, and it's not awful. It should be outta there ns.